src/Http/Controllers/Web/ContactController.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tipoff\Forms\Http\Controllers\Web;
+
+use Illuminate\Routing\Controller;
+use App\Models\Location;
+use App\Models\Market;
+use DrewRoberts\Media\Models\Image;
+use Illuminate\Http\Request;
+
+class ContactController extends Controller
+{
+    public function __invoke(Request $request, Market $market = null, Location $location = null)
+    {
+        $current = getCurrentMarket($request);
+
+        // Redirect to the user's current market if they have one set and end up on page in company section
+        if ($current && $request->segment(1) === 'company') {
+            return redirect()->to($current->contact_path);
+        }
+
+        $image = Image::find(40)->url;
+
+        // Show the page in the company section of the site
+        if ($request->segment(1) === 'company') {
+            return view('website.pages.company.choose', [
+                'market' => $current,
+                'html' => null, // set to true if need to use html page instead of AMP
+                'title' => 'Contact Us',
+                'subtitle' => 'Please select your location from the dropdown below:',
+                'cta' => null,
+                'link' => 'contact',
+                'seotitle' => 'Contact Us at The Great Escape Room',
+                'seodescription' => 'Contact Us at The Great Escape Room, a leader in the escape room industry with 12 locations. Visit our website to learn more.',
+                'ogtitle' => 'Contact Us at The Great Escape Room',
+                'ogdescription' => 'Contact Us at The Great Escape Room, a leader in the escape room industry with 12 locations. Visit our website to learn more.',
+                'canonical' => route('contact'),
+                'image' => $image,
+                'ogimage' => $image === null ? url('img/ogimage.jpg') : $image,
+            ]);
+        }
+
+        // Send to correct market if somehow they get to a URL with a different market than where the location belongs
```

Comment:
@drewroberts I'm assuming this is associated with https://github.com/tipoff/locations/issues/85?